GeoEntity.md

```diff
@@ -0,0 +1,280 @@
+	- Request all administrative area twos.
+	- Request all localities.
+	- Request all postal codes.
+	- Request all metropolitan statistical areas.
+	
+This service supports both HTTP/GET and POST methods.
+All responses with an HTTP status of 200 will consist of a JSON object with a top-level "data" node containing a node "entities", which is an list of entities as requested.
+
+&nbsp;
+----------------------------
+#All Countries Request
+- Request Structure:
+	- Target: http://localhost:8080/admin_area_1s (After adding to Routing layer, we update this)
+	- URL parameter: none
+	- Example URL:http://localhost:8080/countries
+	
+- Response Sample:
+```
+{
+  "data": {
+    "entities": [
+      {
+        "name": "Afghanistan",
+        "country_code": "AF",
+        "place_id": "ChIJbQL_-LZu0TgReNqWvg1GtfM",
+        "location": {
+          "lat": 33.93911,
+          "lng": 67.709953
+        },
+        "location_type": "APPROXIMATE",
+        "viewport": {
+          "northeast": {
+            "lat": 38.4908389,
+            "lng": 74.8898619
+          },
+          "southwest": {
+            "lat": 29.3772319,
+            "lng": 60.5170005
+          }
+        },
+        "formatted_address": "Afghanistan"
+     }
+  ]
+ }
+}
+```
+&nbsp;
+----------------------------
+#All Administrative Area Ones Request
+- Request Structure:
+	- Target: http://localhost:8080/admin_area_1s (After adding to Routing layer, we update this)
+	- URL parameter: 
+  
+    | Parameter  | Required | Description |
+    |----------------|----------------|-----------------|
+    | country        | Yes | A country name or two-letter ISO-3166 country code, not casing sensitive            | 
+    
+	- Example URL: http://localhost:8080/admin_area_1s?country=us
+	
+- Response Sample:
+```
+{
+  "data": {
+    "entities": [
+      {
+        "long_name": "Alaska",
+        "short_name": "AK",
+        "level": 1,
+        "place_id": "ChIJG8CuwJzfAFQRNduKqSde27w",
+        "location": {
+          "lat": 64.2008413,
+          "lng": -149.4936733
+        },
+        "location_type": "APPROXIMATE",
+        "viewport": {
+          "northeast": {
+            "lat": 71.347066,
+            "lng": -129.9945563
+          },
+          "southwest": {
+            "lat": 51.220011,
+            "lng": 172.4573429
+          }
+        },
+        "formatted_address": "Alaska, USA"
+      }
+    ]
+ }
+}
+```
+&nbsp;
+----------------------------
+#All Administrative Area Twos Request
+- Request Structure:
+	- Target: http://localhost:8080/admin_area_2s (After adding to Routing layer, we update this)
+	- URL parameter: 
+  
+    | Parameter  | Required | Description |
+    |----------------|----------------|-----------------|
+    | country        | Yes | A country name or two-letter ISO-3166 country code, not casing sensitive            | 
+    | admin_area_one | Optional  | A first order administrative subdivision, both long name and short name are accepted|
```

Comment:
it might be a lot easier to just accept codes in these fields (for all endpoints, not just this one). otherwise we are vulnerable to complaints about no spelling corrections, etc.